app/models/course.rb

```diff
@@ -619,6 +620,34 @@ def validate_subject_consistency
     end
   end
 
+  def validate_custom_age_range
+    # This regex checks that a 1 or 2 digit number is selected for the 'to' and 'from' values.
+    # It also checks that the age range is correctly formatted with _to_ in between these values
+    valid_regex_pattern = /^(\d{1,2})_to_(\d{1,2})$/
```

Comment:
Regex can be hard to parse, consider splitting it into chunks? Like `/^#{two_digit_number}_to_#{two_digit_number}$/`

Reply:
Pretty sure you can't split a Regex like that?